src/REACTION/fix_bond_react.h

```diff
@@ -55,7 +51,7 @@ class FixBondReact : public Fix {
   double compute_vector(int) override;
   double memory_usage() override;
 
- private:
+ protected:
```

Comment:
Since there is no fix bond/react/kk here, these changes should be reverted.

Reply:
done